mrmustard/physics/representations.py

```diff
@@ -97,6 +100,15 @@ def reorder(self, order: tuple[int, ...] | list[int]) -> Representation:
         Reorders the representation indices.
         """
 
+    @abstractmethod
+    def serialize(self) -> dict[str, ArrayLike]:
```

Comment:
Does it make sense to rename these to `to_dict` and `from_dict` instead?

Reply:
I think actually it would. It looks like the use case is to call `serialize` on the circuit itself. Renaming would made that workflow clearer I believe

Reply:
agreed, done!